src/ui/pages/Landingpage/components/UserNameText.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import { useAuth0 } from '@auth0/auth0-react';
+import { Box } from '../../../components/Base/Base';
+
+export default function UserNameText() {
+    const { user } = useAuth0();
+    return (
+        <Box
+            as="button"
```

Comment:
filename "UserNameText" but its a Button. please rename to "UserNameButton" or change "as=button".